bin/wp-enforcer

```diff
@@ -6,6 +6,36 @@ readonly BIN="$(cd "$(dirname "$0")" && pwd -P)"
 readonly DIR="$BIN/$(dirname "$(readlink "$0")")"
 readonly PROJECT="$(pwd)"
 
+# Define command-line options here.
+# Follow -o with a comma-separated list of single-character option names.
+# Follow -l with a comma-separated list of long option names.
+# Options may be followed by one colon to indicate they have a required argument
+if ! options=$(getopt -o h -l help -- "$@")
+then
+    # something went wrong, getopt will put out an error message for us
+    exit 1
+fi
+
+eval set -- $options
```

Comment:
@daveross It appears this line is causing it to override the passed in options. Commenting out the line allows displaying the help text correctly on a mac.